src/js/controller/piskel/PublicPiskelController.js

```diff
@@ -17,7 +17,6 @@
     this.resetWrap_('setCurrentFrameIndex');
     this.resetWrap_('selectNextFrame');
     this.resetWrap_('selectPreviousFrame');
-    this.resetWrap_('setCurrentLayerIndex');
```

Comment:
We can do it in the same PR, having a dedicated commit will make it easier to review and to go back to. 

This commit should contain:
- removal of the reset wrapper for selectLayer 
- removal of the reset wrapper for setCurrentLayerIndex
- handling the selected layer classname by DOM manipulation instead of templating

Does that sound ok?